RealmTasks Apple/RealmTasks macOS/ListViewController.swift

```diff
@@ -141,16 +138,31 @@ final class ListViewController<ListType: ListPresentable where ListType: Object>
         }
     }
 
+    // MARK: UI Writes
+
+    private func beginUIWrite() {
+        list.realm?.beginWrite()
+    }
+
+    private func commitUIWrite() {
+        try! list.realm?.commitWrite(withoutNotifying: [notificationToken!])
+    }
+
+    func uiWrite(@noescape block: () -> ()) {
+        beginUIWrite()
+        block()
+        commitUIWrite()
+    }
+
     // MARK: Actions
 
     @IBAction func newItem(sender: AnyObject?) {
         endEditingCells()
 
-        try! list.realm?.write {
+        uiWrite() {
```

Comment:
minor style quibble, we don't use empty parens with trailing closure syntax elsewhere in the code: `uiWrite {`